shared/tracker/render.desktop.js

```diff
@@ -35,15 +35,21 @@ export default class Render extends Component {
   }
 
   render2 (styles: Object) {
+    // We have to calculate the height of the footer.
```

Comment:
? confused about this. i don't think this is accurate. it's positioned absolute to be on the bottom, but it can use flexbox to decide its own height based on its contents (which is what we want). you shouldn't need to calculate anything here. what am i missing? @MarcoPolo